ixmp/utils.py

```diff
@@ -147,11 +146,16 @@ def pd_write(df, f, *args, **kwargs):
         writer.save()
 
 
-def numcols(df):
```

Comment:
It's fine to delete/rename this, but we've been bitten before when code in message_ix depends on such methods. Please double-check that nothing in message_ix depends on this.

Reply:
I guess @danielhuppmann checked this, seems to be fine https://github.com/iiasa/message_ix/search?q=numcols&unscoped_q=numcols

Reply:
I did a search before making the change and updated the one other use of this function n the cli.